cli/Sources/TuistCache/Services/SaveCacheCASService.swift

```diff
@@ -86,6 +88,12 @@ public final class SaveCacheCASService: SaveCacheCASServicing {
             case let .json(error):
                 throw SaveCacheCASServiceError.badRequest(error.message)
             }
+        case .requestTimeout:
+            throw SaveCacheCASServiceError.saveFailed
+        case .contentTooLarge:
+            throw SaveCacheCASServiceError.saveFailed
```

Comment:
`saveFailed` is reserved to an unknown error – but we know the reasons here, so we should have instead explicit error cases.